pulp_rpm/tests/functional/constants.py

```diff
@@ -18,7 +18,7 @@
 
 
 RPM_FIXTURE_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm/')
-RPM_FIXTURE_COUNT = 35
+RPM_FIXTURE_COUNT = 39
```

Comment:
Suggestion: maybe we should add a ```get_content_units_by_type()``` function instead, which returns a dictionary instead of a list.  Currently the tests use a ```get_content_units()``` function which is probably too simplistic/generic for multi-content plugins like RPM.

Reply:
I can add a note like this now.